CONTRIBUTING.md

```diff
@@ -1,7 +1,15 @@
 Contributing
 ============
 
-After cloning git repo run (from project root):
+After cloning git repo
+
+run (from `ts` folder):
```

Comment:
To me, this could be better. It would be nice to have `typings.json` at the project root instead of in the `ts` directory, but if you like to have all things related to types in the `ts` folder, this is the way.